lang/en/tool_mfa.php

```diff
@@ -84,3 +84,9 @@
 $string['state:neutral'] = 'Neutral';
 $string['totalweight'] = 'Total weight';
 $string['weight'] = 'Weight';
+$string['timecreated'] = 'Setup time';
+$string['userid'] = 'User ID';
+$string['mfareports'] = 'MFA reports';
+$string['factorreport'] = 'All factor report';
+$string['tableheader'] = 'Auth \ Factor';
```

Comment:
Nit: This is one of those very rare times when I'd suggest this is actually 2 strings and the \ is fixed in code. Both strings could probably be reused from elsewhere. This top left corner could just as easily be empty too